### ISSUE TITLE
[Feature Request] add support for `application/vnd.holoviews_load.v0+json`

### ISSUE THREAD

**Issue by awray3** [benlubas/molten-nvim#142]
Hello, 

I am trying to run `df.plot()` on a pandas Dataframe and view the plot with Molten.
Only `text/plain` and `text/html` options are returned by `MoltenEvaluateLine`, and `:MoltenOpenInBrowser` shows a blank page.

Is there a specific way I should be running plotting code in order for this to work?

**Comment by benlubas** [comment]
If there's a `text/html` output it should work. I can check it out

**Comment by awray3** [comment]
Here's the output of debug logs:
<img width="623" alt="image" src="https://github.com/benlubas/molten-nvim/assets/46057140/1ac0835e-44f4-4e1d-911c-c16907891a0c">

No image is displayed either, even though I have image.nvim installed and configured correctly (I think) with all optional dependencies in my neovim environment (which is separate from the kernel environment)

**Comment by benlubas** [comment]
I'm curious about the HTML mimetype b/c I don't see it. I am seeing the image though. I guess you haven't setup image.nvim correctly

<img width="683" alt="image" src="https://github.com/benlubas/molten-nvim/assets/56943754/1b6a641b-1b6f-4311-be0b-8f4b9027de0e">

**Comment by awray3** [comment]
I can confirm I have image.nvim working (I put the neovim logo there with `require('image').from_file(...)`) and I've set the image.nvim config. I use Lunarvim for the base of my config, perhaps there's a conflict in some autogroup somewhere.

<img width="641" alt="image" src="https://github.com/benlubas/molten-nvim/assets/46057140/88f272f7-b490-40ee-801b-d6a334ae4b6b">

**Comment by benlubas** [comment]
I highly doubt that it's Lunarvim interfering.

do you have you config on github that you could link for me to take a look at?

**Comment by awray3** [comment]
Yes, here's [my config](https://github.com/awray3/dotfiles-lunarvim/blob/main/.config/lvim/config.lua). I'm running on kitty.

**Comment by awray3** [comment]
Aha, I found what was missing from the init function:
```
vim.g.molten_image_provider = "image.nvim"
```
Now plots that come back as `image/png` work!

Though I guess there's still the original problem. I often use [hvplot](https://hvplot.holoviz.org/) which returns 'text/html' instead of 'image/png', and I'm still getting nothing from `:MoltenOpenInBrowser`.

**Comment by benlubas** [comment]
I also see nothing when trying to run the following code:

```python
import hvplot.pandas  # noqa
from bokeh.sampledata import iris

iris = iris.flowers

hvplot.plotting.andrews_curves(
    iris,
    class_column='species',
    samples=20,

```

There are a few hints in the browser console:
![image](https://github.com/benlubas/molten-nvim/assets/56943754/47b1d2b2-e0f4-4963-868b-d560a331ad8f)

And that in conjunction with this from their website:
![image](https://github.com/benlubas/molten-nvim/assets/56943754/34d67cf0-f589-4b49-bfd8-3c1eac9d8f44)


Makes me think that we need to do something extra for this to work...

**Comment by benlubas** [comment]
This is the special mimetype that we could support: `application/vnd.holoviews_load.v0+json`. I'm not sure which javascript library we need to load, or where to get it from or anything like that honestly. Do you have any idea where to look?

**Comment by awray3** [comment]
I could look into it
